webapp-django/crashstats/settings/base.py

```diff
@@ -51,7 +51,7 @@ def path(*dirs):
 
 SITE_ID = 1
 
-LANGUAGE_CODE = "en-US"
```

Comment:
The default value is "en-us", so I figured I'd just remove this line altogether.